cypress/e2e/specialFeatures/liteSiteWeight/helpers/getTotalPageSize.js

```diff
@@ -11,11 +13,12 @@ export default requests => {
         totalSize += contentLength;
       } else {
         getPageSizeInKB(url).then(size => {
+          requestSizes.push({ url, size });
```

Comment:
Can we also include urls which already had content-length (i.e. do this in the if statement above too)